src/routes/+page.svelte

```diff
@@ -3,7 +3,7 @@
 </script>
 
 <svelte:head>
-    <title>Voycar</title>
+    <title>Voycar - Startseite</title>
```

Comment:
Das hätte ich gerne konsistent mit dem entsprechenden Tab in der Navigation, der aktuell noch "Home" heißt. Ich glaube "Startseite" ist besser, also bitte den Tab umbenennen.

Reply:
done